test/ngSanitize/sanitizeSpec.js

```diff
@@ -280,6 +280,29 @@ describe('HTML', function() {
     expect(doc).toEqual('<p><img src="x"></p>');
   }));
 
+  describe('Custom white-list support', function() {
+    beforeEach(module(function($sanitizeProvider) {
+      $sanitizeProvider.addValidElements({
+        inlineElements: ['button'],
```

Comment:
We need tests for all supported categories (e.g. `blockElements`, `svgElements`).